scalding-core/src/main/scala/com/twitter/scalding/typed/memory_backend/MemoryBackend.scala

```diff
@@ -301,6 +302,7 @@ class MemoryWriter(mem: MemoryMode) extends Writer {
 
   def plan[T](m: Memo, tp: TypedPipe[T]): (Memo, Op[T]) =
     m.plan(tp) {
+      implicit val q: Quoted = Quoted.internal
```

Comment:
similar to above, can we not keep dummy implicit Quoted around. I fear we won't know where we are actually sending them. I prefer to be explicit with the dummies (and add a comment to that effect).

Reply:
fixed